winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -699,19 +699,18 @@ static BOOL kerberos_rd_tgt_token(const sspi_gss_data* token, char** target, krb
 		if (len == 0)
 			return TRUE;
 
-		buf = malloc(len);
+		buf = calloc(len, sizeof(char));
 		if (!buf)
```

Comment:
`calloc()` not needed, the job is already done correctly with the usage of `stpcpy` and `*buf = 0`

Reply:
reverted, https://github.com/FreeRDP/FreeRDP/pull/10642 does change stuff here too.